src/ServicePulse.Host/app/js/views/pending_retries/controller.js

```diff
@@ -0,0 +1,375 @@
+﻿/// <reference path="service.js" />
+;
+(function(window, angular, undefined) {
+    "use strict";
+
+    function controller(
+        $scope,
+        $timeout,
+        $location,
+        $moment,
+        $filter,
+        scConfig,
+        toastService,
+        sharedDataService,
+        notifyService,
+        serviceControlService,
+        endpointsService, 
+        redirectService,
+        pendingRetryService) {
+
+        var vm = this;
```

Comment:
It is indented 1 time from the level of `function controller`. Default indent in VS.